### PR TITLE
Prep 2.10.0

### PR DESCRIPTION
**Pull request by makarandp0**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,84 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Support for 1.x will cease on December 4th, 2020**. This branch will only receive fixes for critical issues until that date. Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) when planning your migration to 2.x. For details on the 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.10.0 (December 10, 2020)
+==========================
+
+New Features
+------------
+
+- twilio-video now allows customizing logger using [loglevel](https://www.npmjs.com/package/loglevel) module. With this feature, logs can now be intercepted at runtime to allow real-time processing of the logs which include but not limited to inspecting the log data and sending it to your own server. (JSDK-2373)
```
Comment (manjeshbhargav) [inline CHANGELOG.md:11]:
```suggestion
- You can now intercept logs generated by twilio-video.js using the [loglevel](https://www.npmjs.com/package/loglevel) module. This allows for real-time processing of the logs which include but not limited to inspecting the log data and sending it to your own server. (JSDK-2373)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,84 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
+
+With this change `ConnectOptions`'s `logLevel` property is now deprecated. You can instead use `logger.setLevel` to set the desired log level.
```
Comment (manjeshbhargav) [inline CHANGELOG.md:13]:
```suggestion
  `ConnectOptions`'s `logLevel` property is now deprecated. You can instead use `logger.setLevel` to set the desired log level.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,84 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
+  }).then(function(room) {
+    room.on('participantConnected', function(participant) {
+      console.log(participant.identity + ' has connected');
+    });
+  }).catch(error => {
+    console.log('Could not connect to the Room:', error.message);
+  });
+  ```
+
+Additionally  `ConnectOptions`'s `eventListener` property is now deprecated. You can listen for the signaling events by intercepting the logger's messages as shown in the example below. (JSDK-2977)
```
Comment (manjeshbhargav) [inline CHANGELOG.md:31]:
```suggestion
  Additionally, `ConnectOptions`'s `eventListener` property is now deprecated. You can listen for the signaling events by intercepting the logger's messages as shown in the example below. (JSDK-2977)
```

File: README.md [hunk 4]
```diff
@@ -62,7 +62,8 @@ Releases of twilio-video.js are hosted on a CDN, and you can include these
 directly in your web app using a &lt;script&gt; tag.
 
 ```html
-<script src="//media.twiliocdn.com/sdk/js/video/latest/twilio-video.min.js"></script>
+<script src="//media.twiliocdn.com/sdk/js/video/releases/2.10.0/twilio-video.min.js"></script>
```
Comment (manjeshbhargav) [inline README.md:65]:
```suggestion
<script src="//media.twiliocdn.com/sdk/js/video/latest/twilio-video.min.js"></script>
```

**Review by manjeshbhargav — APPROVED** [review]
@makarandp0 I have requested some small changes. Also, I don't think you need to create a new branch to merge from develop to master. From next time, please open a PR straight from develop to master.

File: README.md (hunk 4 above)
Comment (makarandp0) [reply README.md:65]: @manjeshbhargav - we only pin docs, not the sdk, so we cannot use `latest` here

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,84 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Support for 1.x will cease on December 4th, 2020**. This branch will only receive fixes for critical issues until that date. Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) when planning your migration to 2.x. For details on the 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
```
Comment (donaltoomey) [inline CHANGELOG.md:3]: Lets say, "Support for 1.x ended on December 4th, 2020."

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,84 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
 
+2.10.0 (December 10, 2020)
+==========================
+
+New Features
+------------
+
+- You can now intercept logs generated by twilio-video.js using the [loglevel](https://www.npmjs.com/package/loglevel) module. This allows for real-time processing of the logs which include but not limited to inspecting the log data and sending it to your own server. (JSDK-2373)
+
+  `ConnectOptions`'s `logLevel` property is now deprecated. You can instead use `logger.setLevel` to set the desired log level.
```
Comment (donaltoomey) [inline CHANGELOG.md:13]: Will this break any existing apps?

Comment (manjeshbhargav) [reply CHANGELOG.md:13]: @donaltoomey Deprecated does not mean it is removed. You can still use it, so it won't break any existing apps.

**Pull request merged**